### ISSUE TITLE
`unicorn/prefer-event-target` false positives with Angular `import { EventEmitter } from '@angular/core';`

### ISSUE THREAD

**Issue by the-ult** [sindresorhus/eslint-plugin-unicorn#1853]
The `unicorn/prefer-event-target` gives false positives on the Angular EventEmitter

```ts

import { EventEmitter } from '@angular/core';

@Component({
...
})
export class MyComponent {
  @Output() changed = new EventEmitter<string>(); 
 // Results in: Prefer `EventTarget` over `EventEmitter`.eslint[unicorn/prefer-event-target](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v43.0.0/docs/rules/prefer-event-target.md)

  ....
}
```

- https://angular.io/api/core/EventEmitter

**Comment by wopian** [comment]
Also conflicts with https://www.npmjs.com/package/eventemitter3 which supports Browser environments too.

```ts
import { EventEmitter } from 'eventemitter3'
```

**Comment by omar-elsayed97** [comment]
Same issue here

**Comment by fregante** [comment]
@wopian I don’t see the logic behind that. The point of the rule is that you don’t need anything but `EventTarget`. This bug report is about the actual conflict with a similarly-named but different api offered by Angular.

**Comment by wopian** [comment]
@fregante it has extra features and certain omissions for performance reasons. Replacing it with EventTarget is not a valid usecase for the apps my org uses it in, so in the same position as the Angular conflict.
